src/App.css

```diff
@@ -0,0 +1,116 @@
+@import url('https://fonts.googleapis.com/css2?family=Lato:ital,wght@0,400;0,700;0,900;1,400;1,700;1,900&family=Montserrat:wght@300;400;500&family=Poppins:wght@400;500;600&family=Roboto+Slab:wght@200;300;400;500&display=swap');
+
+#root {
+  width: 414px;
+  border: 2px solid black;
+  border-radius: 10px;
+}
+
+.header {
+  font-family: 'Montserrat', sans-serif;
+  background-color: #d2691e;
+  color: white;
+  border-radius: 5px 5px 0 0;
+}
+
+.container {
+  border-radius: 5px;
+  border: 1px solid black;
+  margin: 15px 15px 15px 15px;
+  padding: 10px;
+  font-family: 'RobotoSlab', sans-serif;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  transition: 0.5s;
+}
+
+.container:hover {
+  transform: scale(1.05);
+}
+
+.calendar {
+  padding-bottom: 10px;
+}
+
+.year {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  font-weight: bold;
+  font-size: 20px;
+  margin: 0;
+}
+
+img {
+  width: 200px;
+  border-radius: 5px;
+}
+
+.logoName {
+  display: flex;
+  align-items: center;
+  gap: 20px;
+  padding: 5px 2px 2px 20px;
+}
+
+h2 {
+  display: flex;
+  text-align: center;
+  margin: 10px;
+  font-size: 1rem;
+  font-family: 'Montserrat', sans-serif;
+}
+
+.calendar,
+.name,
+.currency {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  margin: 0;
+}
+
+.checkLink {
+  margin-left: 64%;
+  border: 1px solid;
+  border-radius: 10px;
+  padding: 3px;
+  color: white;
+}
+
+#container2022 {
+  background-color: #d2691e;
+  color: white;
+}
+
+#container2021 {
+  background-color: #d2691e;
+  color: white;
+}
+
+#container2020 {
+  background-color: #d2691e;
+  color: white;
+}
+
+#container2019 {
+  background-color: #d2691e;
+  color: white;
+}
+
+#container2018 {
+  background-color: #d2691e;
+  color: white;
+}
+
+.Details {
+  display: flex;
+  flex-direction: column;
+  padding: 10px;
+  color: white;
+}
+
+.nav-link {
+  padding-top: 20px;
+}
```

Comment:
- Please ensure your design match that of the original Behance design provided.
Your Design | Original design
![Screenshot from 2023-03-23 12-29-52](https://user-images.githubusercontent.com/44624138/227190726-3bafb92c-b0c8-4c03-9838-37c5ef8ecdc3.png) | 
![Screenshot from 2023-03-23 12-30-07](https://user-images.githubusercontent.com/44624138/227190779-2aa43de0-e17b-4266-ab50-9cc672790148.png)
As seen in the original design, the financial report is supposed to be paired beside each other 👍🏽.

Reply:
Hello @rotimiazeez, could you be more specific? Are u talking about the color? or the component's layout? let's have a zoom call for more clarification, please.